src/datasources/data-frame/DataFrameDataSource.ts

```diff
@@ -45,32 +31,39 @@ export class DataFrameDataSource extends DataSourceBase<DataFrameQuery> {
 
   baseUrl = this.instanceSettings.url + '/nidataframe/v1';
 
-  defaultQuery = defaultQuery
+  defaultQuery = defaultQuery;
 
-  async runQuery(query: DataFrameQuery, { range, scopedVars, maxDataPoints }: DataQueryRequest): Promise<DataFrame> {
+  async runQuery(query: DataFrameQuery, { range, scopedVars, maxDataPoints }: DataQueryRequest) {
```

Comment:
Are we purposely not specifying a return type here?

Reply:
Oh, that wasn't intentional. I had removed it when I was prototyping different result formats.